examples/text_to_image/train_text_to_image_lora.py

```diff
@@ -41,7 +41,7 @@
 import diffusers
```

Comment:
Missing example refactor from this PR: https://github.com/huggingface/diffusers/pull/5331